cms/templates/js/login.html

```diff
@@ -0,0 +1,17 @@
+<div class="wrapper-content wrapper">
+    <form id="login_form" method="post" action="login_post" onsubmit="return false;">
+        <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf }"/>
+        <input id="email" type="email" name="email" placeholder="${_('example: username@domain.com')}"/>
+        <input id="password" type="password" name="password"/>
+
+        <div class="form-actions">
+            <button type="submit" id="submit" name="submit" class="action action-primary">${_("Sign In to
+                {studio_name}").format(studio_name=settings.STUDIO_NAME)}
+            </button>
+        </div>
+
+        <!-- no honor code for CMS, but need it because we're using the lms student object -->
+        <input name="honor_code" type="checkbox" value="true" checked="true" hidden="true">
+    </form>
+</div>
+<script src="cms/include/js/factories/login.js"></script>
```

Comment:
@adampalay so i am adding login.js here so as in events for dom elements are already define [login.js](https://github.com/edx/edx-platform/blob/062e979e0ed14161d0f882091a06e032ca76bb3e/cms/static/js/factories/login.js#L23) but they not binding acutally.